src/vueWrapper.ts

```diff
@@ -104,8 +104,11 @@ export class VueWrapper<T extends ComponentPublicInstance>
   ): DOMWrapper<SVGElementTagNameMap[K]>
   find<T extends Element>(selector: string): DOMWrapper<T>
   find(selector: string): DOMWrapper<Element> {
-    // force using the parentElement to allow finding the root element
-    const result = this.parentElement.querySelector(selector)
+    let result = this.parentElement['__vue_app__']
```

Comment:
nit: that should still be a const

Reply:
Would you mind adding a comment on why we have to do this (you can even add a link to your issue)?

Reply:
this is indeed my mistake. I have modified and resubmitted the code